lib/api/api_auth.go

```diff
@@ -37,6 +37,12 @@ func emitLoginAttempt(success bool, username, address string, evLogger events.Lo
 	}
 }
 
+func unauthorized(w http.ResponseWriter) {
+	time.Sleep(time.Duration(rand.Intn(100)+100) * time.Millisecond)
```

Comment:
Given you already factor out all these small utility functions, might also make this `waitOnUnsuccessfulAuth` or something.

Reply:
Sure!

Reply:
Fixed in 6647b7725da7522372dd78dc3ab7c380ad961eb0.